README.md

```diff
@@ -152,6 +152,9 @@ $ systemctl restart atomic-openshift-master
 
 Create the MIQ template for deployment and verify it is now available in your project
 
+If you wish to add a SSL certificate now, you can edit the application template and provide that now.  Check the Edge Termination section of [Secured Routes](https://docs.openshift.com/container-platform/latest/architecture/core_concepts/routes.html#secured-routes) for more information on that.
```

Comment:
I think the Origin link is probably better here https://docs.openshift.org/latest/architecture/core_concepts/routes.html#secured-routes